demos/simple/seahorn/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Isn't this the same example that we already have?

How hard would it be to make it work for both KLEE and SeaHorn?